source/uwp/SharedRenderer/lib/pch.h

```diff
@@ -22,6 +22,7 @@
 #define FACILITY_ADAPTIVECARDS 0xADA
 #define ERRORBASE_ADAPTIVECARDS 0x1000
 #define E_PERFORM_FALLBACK MAKE_HRESULT(1, FACILITY_ADAPTIVECARDS, ERRORBASE_ADAPTIVECARDS)
+#define E_FALLBACK_NOT_FOUND MAKE_HRESULT(2, FACILITY_ADAPTIVECARDS, ERRORBASE_ADAPTIVECARDS)
```

Comment:
@beervoley First time creating an hresult, is there anything else I need here?

Reply:
nope, I think this is fine :)